adminSiteClient/GrapherConfigGridEditor.tsx

```diff
@@ -1162,7 +1465,13 @@ export class GrapherConfigGridEditor extends React.Component<GrapherConfigGridEd
         const { grapherElement } = this
         return (
             <div className="preview">
-                <h3>Interactive grapher preview</h3>
+                <h5>Interactive grapher preview</h5>
+                <Toggle
+                    label="Keep entity selection"
```

Comment:
Could this label be explained a bit? The `title` below is helpful, but not visible in the UI